#Request
Header header
int16 control_action  #0 --> Lane Keep // 1 --> Left Lane Change // 2 --> Right Lane Change
float32 acc  # lane keeping acceleration
bool doneRLflag   #flag for ending the RL

---
#Response
bool RLActionresult       #0 --> Infeasible action -- 1--> finished action
float32 RLActionTime        #Storing the time taken by the execution of any action, the goal has to be finished